hs_core/models.py

```diff
@@ -3504,6 +3504,57 @@ def get_hs_term_dict(self):
 
         return hs_term_dict
 
+    def replaced_by(self):
+        """ return a list or resources that replaced this one """
+        from hs_core.hydroshare import get_resource_by_shortkey, current_site_url   # prevent import loop
+        print("replacements for {}:".format(self.short_id))
+        replacedby = self.metadata.relations.all().filter(type='isReplacedBy')
+        rlist = []
+        for r in replacedby:
+            replacement = r.value
+            print("   {}:".format(replacement))
+            # TODO: This is a mistake. This hardcodes the server on which the URI is created as its URI
+            if replacement.startswith(current_site_url() + "/resource/"):
```

Comment:
This is something you brought up with bag ingestion as well.  I'm not sure what the correct solution is.  I see your point, but if we're going to move forward with metadata being stored in the rdf/xml, then they must include the full url, because it's rdf.  I've gotten around the different site_urls in the hs_rdf python client by only looking at the paths and using the site url provided by the user when authenticating.

Reply:
I can't afford to change this now.... something for later..... issue generated.